toolchain/temp_dir.go

```diff
@@ -0,0 +1,38 @@
+package toolchain
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"sourcegraph.com/sourcegraph/srclib"
+)
+
+// TEMPDIRNAME is directory under SRCLIBPATH where to store temp directories for toolchains.
+const TEMPDIRNAME = "tmp"
```

Comment:
- [ ] We don't use all-caps for constants (see https://sourcegraph.com/github.com/MaikuMori/srclib@540440047bb78d0fd3a0a3e6b295f67c61635fc9/.tree/buildstore/store.go#def=/github.com/MaikuMori/srclib@master/.GoPackage/github.com/MaikuMori/srclib/buildstore/.def/BuildDataDirName&startbyte=336&endbyte=370). TempDirName works.